hackernews_scraper/src/services/articleSummarizer.ts

```diff
@@ -48,28 +40,25 @@ const createArticleSummarizer = (
       content.length > MAX_CONTENT_LENGTH
         ? '\n[Truncated for length constraints]'
         : '';
-    const emotionalDirective =
-      EMOTIONAL_STIMULI[Math.floor(Math.random() * EMOTIONAL_STIMULI.length)];
 
-    // Build prompt for summary
     const prompt = `
-SUMMARY REQUEST
----------------
-INSTRUCTIONS:
-- ${emotionalDirective}
-- Provide a clear, concise summary of the Hacker News article.
-- Emphasize key technical details, context, and innovative ideas.
-- Do NOT preface with generic phrases such as "The article discusses..."
-- Return ONLY a JSON object with a "summary" key containing 
-  the factual summary. Format must be: { "summary": "..." }.
-
-ARTICLE:
---- TITLE ---
-${sanitizeInput(title)}
-
---- TRUNCATED CONTENT ---
-${sanitizeInput(truncatedContent)} ${truncationNotice}
-`;
+      SUMMARY REQUEST
+      ---------------
+      INSTRUCTIONS:
+      - Provide a clear, concise summary of the Hacker News article.
+      - The summary should be **2 to 3 sentences** long (approximately 50–70 words) and capture the core idea of the article.
+      - Write in a clear, factual style suitable for a tech-savvy audience; assume the reader wants a quick, informative gist.
+      - Highlight the main point and any important outcome or insight, while omitting trivial details or general background.
+      - The tone should be neutral and informative.
+      - Ensure the summary can stand on its own and remains within the optimal length for easy reading on both desktop and mobile.
+
+      ARTICLE:
+      --- TITLE ---
+      ${sanitizeInput(title)}
```

Comment:
I believe xml format is better for prompt adherence. So newline then "<article>" then newline, is what I would recommend